[2022-02-09 08:59:41,669] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-01-01T23:00:00+00:00 [queued]>
[2022-02-09 08:59:41,687] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-01-01T23:00:00+00:00 [queued]>
[2022-02-09 08:59:41,687] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 08:59:41,687] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 08:59:41,688] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 08:59:41,708] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2021-01-01 23:00:00+00:00
[2022-02-09 08:59:41,716] {standard_task_runner.py:52} INFO - Started process 295 to run task
[2022-02-09 08:59:41,730] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'local_to_gcs_task', 'scheduled__2021-01-01T23:00:00+00:00', '--job-id', '476', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingestee_gcs_dag.py', '--cfg-path', '/tmp/tmpdj4c81ds', '--error-file', '/tmp/tmpz2w0yjbk']
[2022-02-09 08:59:41,732] {standard_task_runner.py:77} INFO - Job 476: Subtask local_to_gcs_task
[2022-02-09 08:59:41,857] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-01-01T23:00:00+00:00 [running]> on host cc021fb126ed
[2022-02-09 08:59:41,945] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 08:59:42,002] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2021-01-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-01-01T23:00:00+00:00
[2022-02-09 08:59:47,886] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 08:59:47,901] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=local_to_gcs_task, execution_date=20210101T230000, start_date=20220209T085941, end_date=20220209T085947
[2022-02-09 08:59:47,927] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 08:59:47,964] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-09 09:15:23,176] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-01-01T23:00:00+00:00 [queued]>
[2022-02-09 09:15:23,212] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-01-01T23:00:00+00:00 [queued]>
[2022-02-09 09:15:23,212] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:15:23,213] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:15:23,213] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:15:23,239] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2021-01-01 23:00:00+00:00
[2022-02-09 09:15:23,248] {standard_task_runner.py:52} INFO - Started process 433 to run task
[2022-02-09 09:15:23,260] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'local_to_gcs_task', 'scheduled__2021-01-01T23:00:00+00:00', '--job-id', '577', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingesteee_gcs_dag.py', '--cfg-path', '/tmp/tmpggy8qr9f', '--error-file', '/tmp/tmpaw1fvpgm']
[2022-02-09 09:15:23,262] {standard_task_runner.py:77} INFO - Job 577: Subtask local_to_gcs_task
[2022-02-09 09:15:23,358] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-01-01T23:00:00+00:00 [running]> on host 75953ff4f3d5
[2022-02-09 09:15:23,411] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:15:23,441] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2021-01-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-01-01T23:00:00+00:00
[2022-02-09 09:15:29,617] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 09:15:29,633] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=local_to_gcs_task, execution_date=20210101T230000, start_date=20220209T091523, end_date=20220209T091529
[2022-02-09 09:15:29,683] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:15:29,720] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
